web/modules/custom/dpl_event/dpl_event.services.yml

```diff
@@ -8,7 +8,7 @@ services:
     arguments: ["eventinstance"]
   dpl_event.price_formatter:
     class: Drupal\dpl_event\PriceFormatter
-    arguments: ["@string_translation"]
+    arguments: ["@string_translation", "@config.factory"]
```

Comment:
Instead of injecting the string translation as a dependency you can alsu use the `StringTranslationTrait`on your service. I think it is more neat. Less work.

Reply:
I'd prefer not to change the existing stuff in this PR - If we want to do that in the future, i think we should check all the places that use it and do it here :)